geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloDataStoreFactory.scala

```diff
@@ -169,9 +169,7 @@ class AccumuloDataStoreFactory extends DataStoreFactorySpi {
       cachingParam
     )
 
-  val pi = getParametersInfo
```

Comment:
Yeah, it was unused... maybe it was there for some side-effect? But it seems fine without it...